src/common/precompiled.h

```diff
@@ -4,5 +4,5 @@
 
 #include "globals.h"
 
-#define forceinline __forceinline
+#define inline inline
```

Comment:
Isn't this is redundant?
The line should probably just be removed.

Reply:
It is. I made this commit by

```
$ find -regex ".*\.cpp\|.*\.h"|xargs sed -i 's/__forceinline/inline/'
$ find -regex ".*\.cpp\|.*\.h"|xargs sed -i 's/forceinline/inline/'
```
Thanks for catching that one up. Will update.

Reply:
Might be reading diffs wrong, but it looks like you got this in vstquartzplug.h but not in precompiled.h.